plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zone-methods.php

```diff
@@ -8,6 +8,45 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
+
+use Automattic\WooCommerce\Internal\Admin\WCAdminAssets;
```

Comment:
Seems like line 12 to 49 doesn't belong to this template file but rather to `WC_Settings_Shipping::zone_methods_screen`